src/components/organisms/CompareModal/CompareModal.tsx

```diff
@@ -34,17 +35,24 @@ export const DiffModal: React.FC<Props> = ({visible, onClose}) => {
     >
       {!diffComparison ? <CompareActionBar /> : <DiffActionBar />}
 
-      <div style={{height: 'calc(100% - 66px)', position: 'relative'}}>
-        <Row>
-          <Col span={11}>
-            <ResourceSetSelector side="left" />
-          </Col>
-          <Col span={2} />
-          <Col span={11}>
-            <ResourceSetSelector side="right" />
-          </Col>
-        </Row>
+      <Row ref={containerRef}>
+        <Col span={11}>
+          <ResourceSetSelector side="left" />
+        </Col>
+        <Col span={2} />
+        <Col span={11}>
+          <ResourceSetSelector side="right" />
+        </Col>
+      </Row>
 
+      <div
+        style={{
+          height: `calc(100% - ${height}px - 66px)`,
+          position: 'relative',
+          overflowY: 'auto',
+          overflowX: 'hidden',
+        }}
```

Comment:
Please ignore all these `style` and layout things while reviewing. I'll come back and fix this properly later I add more functionality such as different operations and filter.